tools/checked-c-convert/NewTyp.h

```diff
@@ -84,6 +98,12 @@ class ArrTyp : public NewTyp {
   std::string mkStr() {
     return ReferentTyp->mkStr() + "* ";
   }
+
+  virtual bool anyChanges() { return false || ReferentTyp->anyChanges(); }
```

Comment:
Why do you have "return false || ..." here?  Can you eliminate the false clause?